devicedefender/include/aws/iotdevicedefender/Config.h.in

```diff
@@ -0,0 +1,11 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#define AWS_IOT_DEVICE_SDK_CPP_V2_IOTDEVICEDEFENDER_VERSION "@FULL_VERSION@"
+#define AWS_IOT_DEVICE_SDK_CPP_V2_IOTDEVICEDEFENDER_VERSION_MAJOR @PROJECT_VERSION_MAJOR@
+#define AWS_IOT_DEVICE_SDK_CPP_V2_IOTDEVICEDEFENDER_VERSION_MINOR @PROJECT_VERSION_MINOR@
+#define AWS_IOT_DEVICE_SDK_CPP_V2_IOTDEVICEDEFENDER_VERSION_PATCH @PROJECT_VERSION_PATCH@
```

Comment:
`PROJECT_VERSION_MAJOR`, `PROJECT_VERSION_MINOR `, and `PROJECT_VERSION_PATCH ` are not set in CMake config, so the corresponding defines are empty. Do we need these defines, even if they're empty?

Reply:
https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION_MAJOR.html

Reply:
Sorry, I had to mention that I checked locally and the defines were actually empty in the generated files.

It seems, when we call `project()` in a service CMakeLists.txt, these 3 variables reset to empty because we don't pass the `VERSION` option:
```
project(IotSecureTunneling-cpp LANGUAGES CXX)    # no VERSION here, reset PROJECT_VERSION_* vars
if (DEFINED SIMPLE_VERSION)
    message("Secure Tunneling version is ${SIMPLE_VERSION}")
    set(PROJECT_VERSION ${SIMPLE_VERSION})      # This one does not set PROJECT_VERSION_* vars for me
endif()
```

Reply:
Updated